### PR TITLE
Title Edit v3.0.3.0

### PR DESCRIPTION
**Pull request by RokasKil**
- 7.1 and API11 update
- When using the "Track player location" feature with "Save Eorzea time" unchecked, the plugin will use the current Eorzea time instead of midnight (thanks @Scrxtchy)
- Added an option to use current Eorzea time instead of a predefined time in presets
- Rewrote world layout change detection hooking to fix a rare crash

Depends on https://github.com/aers/FFXIVClientStructs/pull/1183 going through, won't work without it

### CODE REVIEW

**Comment by goaaats** [comment]
bleatbot, rebuild

**Comment by reiichi001** [comment]
I've added the block label so PAC know to not merge this until the CS update goes through. Alternatively, please mark this as a draft and then switch it back to a normal PR once the dependency update is merged and released.

**Comment by RokasKil** [comment]
Marked it as a draft

**Comment by reiichi001** [comment]
I see the CS PR has been merged. If you can confirm that the latest Dalamud release contains the contents, then I can work on reviewing this and merging it.

**Review by reiichi001 — APPROVED** [review]

**Pull request merged**
